go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,25 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+	filePos, err := strconv.Atoi(resultMap["Exec_Master_Log_Pos"])
+	if err != nil {
+		return SlaveStatus{}, fmt.Errorf("invalid FilePos GTID (%v): expecting pos to be an integer", resultMap["Exec_Master_Log_Pos"])
+	}
+
+	status.FilePosition.GTIDSet = filePosGTID{
+		file: resultMap["Relay_Master_Log_File"],
+		pos:  filePos,
+	}
+
+	fileRelayPos, err := strconv.Atoi(resultMap["Relay_Log_Pos"])
+	if err != nil {
+		return SlaveStatus{}, fmt.Errorf("invalid FilePos GTID (%v): expecting pos to be an integer", resultMap["Exec_Master_Log_Pos"])
+	}
+
+	status.FileRelayLogPosition.GTIDSet = filePosGTID{
+		file: resultMap["Relay_Log_File"],
```

Comment:
Similar to above, this should be `Master_Log_File`: the name of the binlog file on the master that the replica's I/O thread is currently processing.

Reply:
Done.